packages/api-client/src/models/manifestPreview.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable max-classes-per-file */
+import { BaseResponse, BaseValidationRequest, BaseValidationResponse } from './base';
+import { ElectionManifest } from './election';
+import { ElementModQ } from './keyCeremony';
+
+export default interface ManifestPreview {
+    name: string;
+    numberOfContests: number;
+    numberOfStyles: number;
+    startDate: Date;
+    endDate: Date;
+    fileHash: string;
+    fileName: string;
+}
+
+export class Manifest {
+    manifest_hash: ElementModQ;
+
+    manifest: ElectionManifest;
+}
+export class ManifestQueryResponse extends BaseResponse {
+    manifests: Manifest[] = [];
+}
+
+export class ValidateManifestRequest extends BaseValidationRequest {
+    /*
```

Comment:
This should follow jsdoc for typescript. This usually sits on top of the class. Similar with the comment below it. 

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
comments updated